client/blocks/subscription-length-picker/README.md

```diff
@@ -1,8 +1,27 @@
-TermPickerOption
+SubscriptionLengthPicker
 ==========
 
-Term picker option component: represents a term that user could pick when choosing his plan
-(monthly, yearly, biennially)
+isplays a list of terms (monthly, yearly, biennially).
+
+### `index.jsx`
+
+Given an array of plans, this component creates a corresponding list of TermPickerOption components to allow
```

Comment:
Just checking — is this suppose to be `TermPickerOption` or `SubscriptionLengthOption`?